### PR TITLE
update change log for 2.0.1

### PR DESCRIPTION
**Pull request by nolouch**

### CODE REVIEW

**Comment by disksing** [comment]
LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # PD Change Log
 
 ## v2.0.1
+### New Feature
+* Add `Scatter Range` scheduler to balance the regions with specify key range
```
Comment (lilin90) [inline CHANGELOG.md]: -> Add the `Scatter Range` scheduler to balance Regions with the specified key range

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +11,7 @@
 * Fix the error that occurs when parsing the configuration file
 * Fix the issue that the etcd leader and the PD leader are not synchronized
 * Fix the issue that Learner still appears after it is closed
+* Fix the issue that load regions failure due to packet size too large
```
Comment (lilin90) [inline CHANGELOG.md]: -> Fix the issue that Regions fail to load because the packet size is too large

**Comment by nolouch** [comment]
PTAL @lilin90

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # PD Change Log
 
 ## v2.0.1
+### New Feature
+* Add the `Scatter Range` scheduler to balance regions with the specified key range
```
Comment (lilin90) [inline CHANGELOG.md]: regions -> Regions

Note: "Region" is a proper noun for the TiDB project, so please capitalize the first letter of "Region" to keep it consistent with its occurrences in other documents.

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +11,7 @@
 * Fix the error that occurs when parsing the configuration file
 * Fix the issue that the etcd leader and the PD leader are not synchronized
 * Fix the issue that Learner still appears after it is closed
+* Fix the issue that regions fail to load because the packet size is too large
```
Comment (lilin90) [inline CHANGELOG.md]: Ditto.

File: CHANGELOG.md (hunk 3 above)
Comment (nolouch) [reply CHANGELOG.md]: got it. done.

**Pull request merged**